KEEP/gettext-gnu-musl.fix.patch

```diff
@@ -0,0 +1,36 @@
+--- a/gettext-runtime/gnulib-lib/stdio.in.h
++++ b/gettext-runtime/gnulib-lib/stdio.in.h
+@@ -141,7 +141,9 @@
+    so any use of gets warrants an unconditional warning.  Assume it is
+    always declared, since it is required by C89.  */
+ #undef gets
++#if HAVE_RAW_DECL_GETS
+ _GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
++#endif
```

Comment:
wouldnt it be better to turn off -Werror instead of adding a patch to mute pedantic warnings ?